### PR TITLE
[COR-137] Create EqualError for RPC Errors that ignore line number

### PR DESCRIPTION
**Pull request by nmarie22**

### CODE REVIEW

File: database/errors_test.go [hunk 1]
```diff
@@ -264,3 +264,79 @@ func TestDatabaseToApiError(t *testing.T) {
...
+			err:      errors.New("foo[GAD.DAT.123]bar"),
+			expected: "foo[GAD.DAT.567]bar",
+			equal:    true,
+		},
+		{
+			name:     "error message mismatch",
+			err:      errors.New("foo[GAD.DAT.123]bar"),
+			expected: "foo[GAD.DAT.567]baz",
+			equal:    false,
+		},
```
Comment (apynes) [inline database/errors_test.go:325]: Missing tests for conditions: 
1. No brackets in the string e.g. "foo bar"
2. Multiple brackets e.g. "foo bar [ABC.DEF.123] [QWE.ZXC.256]"

It appears that they should pass when you add them.

**Review by apynes — APPROVED** [review]
LGTM just missing a couple of test cases.

File: database/errors_test.go (hunk 1 above)
Comment (apynes) [reply database/errors_test.go:325]: Also missing empty string tests, e.g. errors.New("")

Comment (nmarie22) [reply database/errors_test.go:331]: Updated, thanks!

**Review by apynes — APPROVED** [review]
LGTM :+1:

**Pull request merged**